packages/esm-admin-openconceptlab-app/src/subscription/subscription.component.tsx

```diff
@@ -59,6 +68,17 @@ const Subscription: React.FC = () => {
       evt.preventDefault();
       evt.stopPropagation();
 
+      if (isSnapshotOptionDisabled && isSubscribedToSnapshot) {
+        showNotification({
```

Comment:
It seems like we could do something with a modal dialog box instead? I.e., "You can't subscribe to a snapshot release if you've subscribed to a specific collection version. Do you want to remain subscribed to a specific version or switch to a snapshot?"

Also, it would be best to change "SNAPSHOT" -> "HEAD". I know the old UI was using SNAPSHOT, but OCL uses HEAD for the same concept, so it would probably be more understandable.

Reply:
> It seems like we could do something with a modal dialog box instead? I.e., "You can't subscribe to a snapshot release if you've subscribed to a specific collection version. Do you want to remain subscribed to a specific version or switch to a snapshot?"

Actually, this error is already prevented by disabling the snapshot checkbox when a version is given with the URL.  So I thought a notification would be enough. If we are displaying a modal, do I have to use the `showModal` from `@openmrs/esm-framework`?

> Also, it would be best to change "SNAPSHOT" -> "HEAD". I know the old UI was using SNAPSHOT, but OCL uses HEAD for the same concept, so it would probably be more understandable.

Shall I rename all the occurrences of `SNAPSHOT` to `HEAD`?